cmd/fs-v1.go

```diff
@@ -181,6 +183,11 @@ func NewFSObjectLayer(fsPath string) (ObjectLayer, error) {
 	return fs, nil
 }
 
+// gets name of meta json file
+func (fs *FSObjects) MetaJSONFile() string {
```

Comment:
This is a superfluous function it's really not needed since we are simply accessible fs.metaJSONFile

Reply:
removed this function.